[DOCS-10298] Update monitor type to CI/CD & Test (#29007)
DataDog/documentation: content/en/monitors/types/ci.md

---

{{< site-region region="gov" >}}

> <div class="alert alert-warning">This feature is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>

Reviewer: ```suggestion
{{< site-region region="gov" >}}
<div class="alert alert-warning">The CI/CD & Test Monitor is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
{{< /site-region >}}
```
Just suggest to add the name of the feature and use the new callout so it links to the Site page

{{< /site-region >}}

## Overview

To create a monitor for CI pipelines, CI tests, or CD deployments, first enable the related product for your organization:

| Monitor type     | Required product         |
|------------------|--------------------------|
| CI Pipeline      | [CI Visibility][1]       |
| CI Test          | [Test Optimization][2]   |
| CD Deployments   | [CD Visibility][3]       |

> CI/CD and test monitors allow you to visualize CI/CD data and set up alerts on it. For example, create a CI Pipeline monitor to receive alerts on a failed pipeline or a job. Create a CI Test monitor to receive alerts on failed or slow tests.

Reviewer: ```suggestion
CI/CD and Test monitors allow you to visualize CI/CD data and set up alerts on it. For example, create a CI Pipeline monitor to receive alerts on a failed pipeline or a job. Create a CI Test monitor to receive alerts on failed or slow tests.
```
I think this should be capitalized throughout? As with it lowercase, it read like it was "testing" something vs the name of the monitor.

4. Group CD Deployment events by multiple dimensions (optional):
    * All CD Deployment events matching the query are aggregated into groups based on the value of up to four facets.
5. Configure the alerting grouping strategy (optional):
   * If the query has a `group by`, multi alerts apply the alert to each source according to your group parameters. An alerting event is generated for each group that meets the set conditions. For example, you could group a query by `@deployment.name` to receive a separate alert for each CD Deployment name when the number of errors is high.

{{< img src="monitors/monitor_types/cd_deployments/define-the-search-query.png" alt="A query for Deployment Status:Error that is being set to group by Deployment Name" style="width:100%;" >}}

#### Using formulas and functions

You can create CD Deployment monitors using formulas and functions. This can be used, for example, to create monitors on the **rate** of an event happening, such as the rate of a deployment failing (error rate).

> The following example is of a pipeline error rate monitor using a formula that calculates the ratio of "number of failed deployment events" (`deployment.status:error`) over "number of total deployment events" (no filter), grouped by `deployment.name` (to be alerted once per deployment). To learn more, see the [Functions Overview][1].

Reviewer: ```suggestion
The following example demonstrates a pipeline error rate monitor. It uses a formula to calculate the ratio of "failed deployment events" (`deployment.status:error`) over "total deployment events" (without filters), grouped by `deployment.name`, to trigger alerts for each deployment individually. To learn more, see the [Functions Overview][1].
```
Just a suggestion for rewording as Vale was flagging the sentence length.

#### Advanced alert conditions

For detailed instructions on the advanced alert options (such as evaluation delay), see the [Monitor configuration][4] page.

### Notifications

For detailed instructions on the **Configure notifications and automations** section, see the [Notifications][5] page.

#### Samples and breaching values top list

> When a CI Pipeline, CI Test, or CD Deployments monitor is triggered, samples or values can be added to the notification message.

Reviewer: Not sure why it's flagging the Oxford comma as you are using it correctly here :)

Author: Maybe Vale thinks that `triggered, samples or values` is a list of things?
